src/pages/recipes/index.md

```diff
@@ -451,3 +451,394 @@ function main(){
     }
 }
 ```
+
+## Events
+InDesign UXP scripting can respond to standard application and document events, such as opening a file, creating a new one, printing, and importing text and graphic files from a disk, etc. Scripts can be attached to events using the eventListener scripting object. Scripts that use events are the same as other scripts except that after the event listener is associated, the script is expected to be blocked by a promise and wait for the event in the background until the event listener is removed. Then the promise can be resolved to exit the script. Once you run the Events Scripts with an event listener, they are called back automatically after the corresponding event occurs, rather than being run by the user (from the Scripts palette).
```

Comment:
Make it future-proof. Remove the reference to 'UXP scripting'.

Reply:
Agreed